app/views/layouts/stories.html.erb

```diff
@@ -2,13 +2,69 @@
 <html lang="en" class="govuk-template">
     <%= render "sections/head" %>
     <%= analytics_body_tag class: "govuk-template__body govuk-body", data: { controller: "video link", target: "link.content" }, id: "body" do %>
-        <%= render "sections/header" %>
-        <main class="content container-1000" role="main" id="main-content">
-            <%= yield %>
-        </main>
-        <%= render "sections/footer" %>
-        <%= render "components/videoplayer" %>
-        <%= render "sections/cookie-acceptance" %>
+      <%= render "sections/header" %>
+      <main class="content container-1000" role="main" id="main-content">
+        <% if @front_matter.dig("build_layout_from_frontmatter") %>
+
+          <div class="content-wrapper">
```

Comment:
Is it worth splitting this into a partial just to reduce the nesting complexity?

Reply:
This is laying the groundwork for the next PR which will move it to a view component and add a load of tests.